documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,91 @@
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+   1. The competition should have at least 12 competitors.
+9. Have at least one designated WCA Delegate in attendance.
+   1. The WCA Delegate for a competition is responsible for:
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Submitting the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be double-checked and free of errors before submission.
+         2. All scramble sequences used during a competition must be sent with the results of the competition.
+            1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups on a round, it is not necessary to identify which competitor was in which group.
+         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained for 7 years.
+      3. Sending corrections of competition results to the WCA Results Team.
+      4. Advising other competition officials when necessary.
+      5. Deciding about disqualifying competitors during the competition.
+      6. Generating, providing, and maintaining the security of the scramble sequences ([WCA Regulation 4b2)](wca{regulations/#4b2})).
+      7. Making a copy of the WCA Regulations available at the competition.
+   2. The WCA Delegate may delegate some of their responsibilities to organizers or other officials.
+
+If WCA Regulations are not correctly observed during a competition, results may be modified at the discretion of the WCA Delegate, WCA Regulations Committee, WCA Results Team, WCA Disciplinary Committee, or the WCA Board.
+
+### Announcement Criteria
+The WCA Delegate must submit the following information when requesting approval of a competition:
+
+1. Competition name, nickname, and ID
+   1. The competition name must abide by the [WCA Competition Name Policy](wca{documents/policies/Competition Name.pdf}).
+   2. The competition nickname and ID must resemble the competition name.
+      1. The competition nickname and ID should only be modified if the competition name is too long (more than 32 characters). Exception: groups of competitions may have modified nicknames and IDs to ensure consistency with past and future competitions.
+2. Location(s)
```

Comment:
I think the proximity policy can fit under "Location(s)", though I think 3x3x3 Fewest Moves Simultaneous Competitions and Twin Competitions might not belong here. I am not sure the best place to describe those.

Reply:
`3. Comply with all [WCA Competition Policies](wca{documents}).`
This line will need to be removed if so.

I feel like FMC competitions would also fit fine in the Locations section.

As well, we should probably define when a competition should be one competition instead of two (or vice versa) I.e. https://www.worldcubeassociation.org/competitions/PBQWaterloo2018 and https://www.worldcubeassociation.org/competitions/No3x3x3Waterloo2018 should have probably been 1 competition with 5km distance, disjoint events, same organizers/delegates, nearly same competitors. But where would we draw the line? WCAT briefly discussed this some time ago and never came to any conclusions about what is appropriate. Some felt that 5 km was too much here: https://www.worldcubeassociation.org/competitions/PeachState2019 So I'm not entirely convinced that there is a clear decision here, but would love to hear thoughts on it.

Reply:
I would keep 3. but removing the "Competition" from it. As for instance Dues Policy must still be observed, I think keeping the link is not a bad idea.

I have no good suggestion on when two competitions should be one :confused: 

By the way, I'm adding the whole Proximity Policy under "Location(s)", I will push the new version now.

Reply:
Perhaps when the organizer(s) & delegate(s) are the same and the competitions are on consecutive days in the same city.